src/agoradatatools/etl/transform/modelAD_general_transform.py

```diff
@@ -0,0 +1,52 @@
+"""
+This module contains the transformation logic for the biomarkers and pathology datasets.
+This is for the Model AD project.
+"""
+
+import pandas as pd
+from typing import Dict, List
+
+
+def modelAD_general_transform(
```

Comment:
I saw the `nest_fields()` function in `utils.py` and considered editing it to fit this use-case while also serving its original purpose. I decided not to do it because it would require more intensive testing and changes that impact a larger amount of existing functionality. Thoughts?

Reply:
I agree with your approach of keeping them separate. In the future if the functionality needed between use cases diverged too much it may be a pain to undo that type of change

Reply:
I agree, I think it's simpler right now to have them separate.